src/Apps/NetPad.Apps.App/appsettings.Development.json

```diff
@@ -1,5 +1,5 @@
 {
-    "Urls": "http://localhost:57940",
+    "Urls": "http://0.0.0.0:57940",
```

Comment:
Binding to `0.0.0.0` here is a security risk if the machine running NetPad is exposed to an untrusted network. Instead we can have the docker script create an `appsettings.Local.json` that overrides that setting and sets it to `0.0.0.0`. NetPad will use a `appsettings.Local.json` file if one exists.